### PR TITLE
Fixed a couple of issues.

### PR DESCRIPTION
**Pull request by saleem-latif**
I am currently testing this, but still can be reviewed.

### CODE REVIEW

File: src/main/java/org/intellij/sdk/codesync/Constants.java [hunk 1]
```diff
@@ -84,5 +84,8 @@ private Notification() {
         public static String INIT_SUCCESS_MESSAGE = "Repo initialized successfully, your code will now be synced with CodeSync.";
         public static String INIT_ERROR_MESSAGE = "Repo initialization errored out, please try again later. If problem persists then contact support.";
         public static String INIT_FAILURE_MESSAGE = "Repo could not be initialized successfully, please try again later. If problem persists then contact support.";
+
+        public static String REPO_IN_SYNC_MESSAGE = "Repo is being synced with CodeSync.";
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/Constants.java:88]: ```Repo pythonproject is in sync with CodeSync.``` i.e. add repo name

File: src/main/java/org/intellij/sdk/codesync/state/PluginState.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package org.intellij.sdk.codesync.state;
 
 public class PluginState {
-    public String value;
+    public boolean isAuthenticated;
+    public boolean isRepoInSync;
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/state/PluginState.java:5]: When isRepoInSync is false, we need to show "Connect you repo with CodeSync" along with Connect button that will trigger the init process (maybe we are doing that already)

Comment (saleem-latif) [reply src/main/java/org/intellij/sdk/codesync/state/PluginState.java:5]: Yes, that is already happening.

Comment (Basit-qc) [reply src/main/java/org/intellij/sdk/codesync/state/PluginState.java:5]: So it becomes something like
```
if (not logged-in) {
	// Show Welcome to CodeSync with Join button
	// Join will trigger Auth process
	return
} else if (repo is not in config.yml) {
	// show "Connect you repo with CodeSync" along with Connect button that will trigger the init process
	return
}

// Show Repo X is in sync with CodeSync, with View on web and Unsync options
```

**Pull request merged**